frontend/src/scenes/sceneLogic.ts

```diff
@@ -106,18 +115,92 @@ interface Params {
 }
 
 interface SceneConfig {
-    onlyUnauthenticated?: boolean // Route should only be accessed when logged out (N.B. should be added to posthog/urls.py too)
-    allowUnauthenticated?: boolean // Route **can** be accessed when logged out (i.e. can be accessed when logged in too; should be added to posthog/urls.py too)
-    dark?: boolean // Background is $bg_mid
-    plain?: boolean // Only keeps the main content and the top navigation bar
-    hideTopNav?: boolean // Hides the top navigation bar (regardless of whether `plain` is `true` or not)
-    hideDemoWarnings?: boolean // Hides demo project warnings (DemoWarning.tsx)
+    /** Route should only be accessed when logged out (N.B. should be added to posthog/urls.py too) */
+    onlyUnauthenticated?: boolean
+    /** Route **can** be accessed when logged out (i.e. can be accessed when logged in too; should be added to posthog/urls.py too) */
+    allowUnauthenticated?: boolean
+    /** Background is $bg_mid */
+    dark?: boolean
+    /** Only keeps the main content and the top navigation bar */
+    plain?: boolean
+    /** Hides the top navigation bar (regardless of whether `plain` is `true` or not) */
+    hideTopNav?: boolean
+    /** Hides demo project warnings (DemoWarning.tsx) */
+    hideDemoWarnings?: boolean
+    /** Route requires project access */
+    projectBased?: boolean
 }
 
 export const sceneConfigurations: Partial<Record<Scene, SceneConfig>> = {
+    // Project-based routes
+    [Scene.Dashboards]: {
+        projectBased: true,
```

Comment:
I wonder if instead we should default to everything being `projectBased` unless you specify otherwise. This feels safer and also most routes will be project-based anyways.